schemas/2015-01-01/Sendgrid.Email.json

```diff
@@ -0,0 +1,115 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2014-04-01-preview/Sendgrid.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Sendgrid.Email",
+  "description": "Sendgrid Resource Types",
+  "resourceDefinitions": {
+    "accounts": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "minLength": 4,
+          "maxLength": 50,
+          "description": "Name of the resource"
+        },
+        "type": {
+          "enum": [
+            "Sendgrid.Email/accounts" 
+          ]
+        },
+        "apiVersion": {
+          "enum": [ 
+            "2015-01-01" 
+          ]
+        },
+        "plan": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "oneOf": [
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                },
+                {
+                  "enum": [
+                    "Free",
+                    "Bronze",
+                    "Silver",
+                    "Gold",
+                    "Platinum",
+                    "Premier"
+                  ]
+                }
+              ],
+              "description": "Plan name"
+            },
+            "publisher": {
+              "type": "string",
+              "description": "Publisher name"
+            },
+            "product": {
+              "oneOf": [
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                },
+                {
+                  "enum": [
+                    "free",
+                    "bronze",
+                    "silver",
+                    "gold",
+                    "platinum",
+                    "2500000"
+                  ]
+                }
+              ],
+              "description": "Plan id"
+            },
+            "promotionCode": {
+              "type": "string",
+              "description": "Promotion code"
+                
+            }
+          },
+          "required": ["name", "product"],
+          "description": "SendGrid plan"
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "password": {
```

Comment:
on the portal.azure.com interface and on our side we have this limitation: password need to have between 8-50 chars